xml_inscriptions/transcribed/NJ.Princ.PU.AM.2003-129A-B.xml

```diff
@@ -7,23 +7,23 @@
         <fileDesc>
```

Comment:
Did you mean to completely get rid of the Fordham file? Which seems to be what happened here. Make sure that you haven't actually gotten rid of that older file before I merge this in.

Reply:
I have no idea what I did here, but I checked in my folders and that Fordham file is still existing...